arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-custom-gas-token-chain.md

```diff
@@ -37,31 +37,11 @@ Deploying a custom gas token Orbit chain is similar to deploying an AnyTrust Orb
 
 ## 1. Custom gas token specification
 
-The difference between custom gas token chains and other Orbit chains is the use of an `ERC-20` token as gas. Enabling this feature requires that you select an existing `ERC-20` token or deploy a new one on the parent chain.
+The difference between custom gas token chains and other Orbit chains is the use of an `ERC-20` token as the gas token. Enabling this feature requires that you select an existing `ERC-20` token or deploy a new one on the parent chain.
 
 ## 2. Chain configuration
 
-You can configure your Orbit chain using the [`prepareChainConfig`](https://github.com/OffchainLabs/arbitrum-orbit-sdk/blob/1f251f76a55bc1081f50938b0aa9f7965660ebf7/src/prepareChainConfig.ts#L3-L31) method and assigning it to a `chainConfig` variable.
-
-Example:
-
-```js
```

Comment:
I think we should leave this snippet here to show where is `nativeToken` specified.
Also, we could leave the table below, or at least copy the description of the `nativeToken` field somewhere.

Reply:
Nvm, just noticed we don't use `nativeToken` in the prepareChainConfig method.